src/components/KonnectorTile.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Link, withRouter } from 'react-router'
+import { getKonnectorIcon } from '../lib/icons'
+import { translate } from 'cozy-ui/react/I18n'
+
+const KonnectorTile = props => {
+  const { icon, konnector, route, router, subtitle, t } = props
+  const { name } = konnector
+  return (
+    <Link className="item-wrapper" to={`${router.location.pathname}/${route}`}>
+      <header className="item-header">
+        <img
+          className="item-icon"
+          alt={t('connector.logo.alt', { name })}
+          src={getKonnectorIcon(konnector)}
+        />
+      </header>
+      <p className="item-title">{name}</p>
```

Comment:
I'm very happy to see you using `alt` on `img` tags, but now I'm disapointed that a `item-title` uses a `p` tag 😛

Reply:
Good point. 👍